synapsemonitor/__main__.py

```diff
@@ -24,7 +24,8 @@ def monitor_cli(syn, args):
         syn_id=args.synapse_id,
         email_subject=args.email_subject,
         users=args.users,
-        days=args.days,
+        value=args.value,
```

Comment:
I'm not a fan of the unit and value parameter here.  I think it should be rate as well.  We can have a function that parses out the unit and value.